app/controllers/admin/people_controller.rb

```diff
@@ -50,12 +50,9 @@ def reorder_role_appointments
 
   def update_order_role_appointments
     current_role_appointment_orderings = @person.current_role_appointments.map(&:ordering)
+    new_ordering = map_ordering_to_current_role_appointment_ordering_values(current_role_appointment_orderings)
 
-    params[:ordering].each do |appointment_row|
-      id, ordering = appointment_row
-      role_appointment = @person.role_appointments.find(id)
-      role_appointment.update!(ordering: current_role_appointment_orderings[ordering.to_i - 1])
```

Comment:
Removing this `update!` causes model callbacks not to trigger which is why the controller test is failing. Before we worry about trying to fix the controller i thought it would be worth discussing a few options:

1. adding functionality to the reorder! method which toggles `update_column` to `update!`
2. manually call republishing all the dependencies. Maybe use a service object or something similar which wraps up calling the 3-4 methods the callbacks do
3. not using the extension in this use case

Let me know what you think!

Reply:
Yes that's exactly the case. I guess we could pass an additional argument like `callbacks: true` which could toggle between the two.

I'm just aware it makes the method slightly more complex. I'll put something together for that now 👍 in a new commit.

Reply:
I'm generally [against boolean flag args](https://martinfowler.com/bliki/FlagArgument.html) in favour of separate methods, but not strongly in this case. I was hoping we could just apply callbacks for everything now without exception